### PR TITLE
Updates + Fixes.

### PR DESCRIPTION
**Pull request by ishangupta-ds**

### CODE REVIEW

**Comment by ishangupta-ds** [comment]
@gdsoumya @amityt

File: app/client/src/App.tsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
-import React, { lazy, Suspense, useEffect } from "react";
-import { Redirect, Route, Router, Switch } from "react-router-dom";
-import { useStyles } from "./App-styles";
-import { Loader } from "./components";
-import withTheme from "./hoc/themeHoc";
-import { useActions } from "./redux/actions";
-import * as AnalyticsActions from "./redux/actions/analytics";
-import * as GithubActions from "./redux/actions/github";
-import { history } from "./redux/configureStore";
+import React, { lazy, Suspense, useEffect } from 'react';
```
Comment (gdsoumya) [inline app/client/src/App.tsx]: Remove these changes only keep the css changes that you have

File: app/client/src/components/CustomCard/CardContent.tsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
-import { Tooltip, Link, Typography, Button } from "@material-ui/core";
-import React from "react";
-import { formatCount } from "../../utils";
-import { CardProps } from "./model";
-import { useStyles } from "./styles";
-import InfoIcon from "@material-ui/icons/Info";
-import clsx from "clsx";
+import { Tooltip, Link, Typography, Button } from '@material-ui/core';
```
Comment (gdsoumya) [inline app/client/src/components/CustomCard/CardContent.tsx]: Unnecessary changes, will make things difficult to debug later. Just send the changes for css that you have

**Review by gdsoumya — CHANGES_REQUESTED** [review]

File: app/client/src/App.tsx (hunk 1 above)
Comment (gdsoumya) [reply app/client/src/App.tsx]: These single quotes need to be converted to double quotes

**Review by gdsoumya — CHANGES_REQUESTED** [review]

**Comment by ishangupta-ds** [comment]
@gdsoumya PTAL now.

**Pull request merged**